src/store/actions/types.js

```diff
@@ -2,3 +2,4 @@ export default {
   SET_ARTICLES: 'SET_ALL_ARTICLES',
   ADD_ARTICLE: 'ADD_ARTICLE',
 };
+export const SIGNUP = 'SIGNUP';
```

Comment:
This works perfectly fine, but for consistency, we could add it to the other types like this.
```js
export default {
  SET_ARTICLES: 'SET_ALL_ARTICLES',
  ADD_ARTICLE: 'ADD_ARTICLE',
  SIGNUP: 'SIGNUP',
};
```
And then update all places that reference it

Reply:
I and you had agreed to leave it there for the meantime since we were exporting differently and work on it sometime when refactoring.